src/htmx.js

```diff
@@ -1141,31 +1141,51 @@ var htmx = (function() {
    */
   function querySelectorAllExt(elt, selector, global) {
```

Comment:
global optional field can be removed with your change as it is never called except by internal recursion which you have removed.  This function is never used elsewhere with this param and it is not published for internalAPI etc.

Reply:
also once you define `let global = false` at the top you no longer need to !! the global's to make the boolean.

Sorry for too many comments :/

Reply:
You're right! I went into "keep it backwards compatible" by habit without even checking usages

No worries for the comments count, especially when it helps improving the code 😛